front/app/containers/App/index.tsx

```diff
@@ -268,7 +268,7 @@ const App = ({ children }: Props) => {
   const { pathname } = removeLocale(location.pathname);
   const urlSegments = location.pathname.replace(/^\/+/g, '').split('/');
   const disableScroll = fullscreenModalEnabled && signUpInModalOpened;
-  const isAuthenticationPending = !authUser && isLoading;
+  const isAuthenticationPending = authUser === undefined;
```

Comment:
I noticed on our demo platform that sometimes when I refresh, I get an infinite spinner and the homepage is not shown. I could not reproduce this in dev mode but I strongly suspect it's linked to our e2e test failures where a bunch of them fail to load the homepage.

I am hoping that by checking is the authUser is undefined (it will be `null` if the user is not logged in) we can resolve the issue.